src/ripple/protocol/impl/SField.cpp

```diff
@@ -150,6 +150,9 @@ CONSTRUCT_TYPED_SFIELD(sfMintedNFTokens,        "MintedNFTokens",       UINT32,
 CONSTRUCT_TYPED_SFIELD(sfBurnedNFTokens,        "BurnedNFTokens",       UINT32,    44);
 CONSTRUCT_TYPED_SFIELD(sfHookStateCount,        "HookStateCount",       UINT32,    45);
 CONSTRUCT_TYPED_SFIELD(sfEmitGeneration,        "EmitGeneration",       UINT32,    46);
+// Three field values of 47, 48 and 49 are reserved for 
+// LockCount(Hooks), VoteWeight(AMM), DiscountedFee(AMM)
+CONSTRUCT_TYPED_SFIELD(sfFirstNFTokenSequence,  "FirstNFTokenSequence", UINT32,    50);
```

Comment:
Ensure that there's no conflict with hooks or other amendments here. Tag @RichardAH. 

Having a registry for this kind of thing could be nice.

Reply:
@RichardAH @gregtatcam 

 I reserved three values of the 32-bit field for what it looks like three new `SFields` in both AMM and Hook. Just want to make sure the number of reserved field value is correct and would be no further changes